Makefile

```diff
@@ -7,11 +7,11 @@ install-deps:
 .PHONY: test
 test:
 	@echo " > \033[32mTesting contracts... \033[0m "
-	truffle test
+	npx truffle test
 
 compile:
 	@echo " > \033[32mCompiling contracts... \033[0m "
-	truffle compile
+	npx truffle compile
```

Comment:
My reason for adding it is because I personally avoid installing packages globally unless I need to, so I can avoid any conflicts with other projects. Using `npx` should allow a user to either use the locally installed package or their global install if it's not found locally.

An alternative to this would be running the locally installed `truffle` directly like it was in the `ChainBridgeV2` repo